### PR TITLE
Learning objectives

### PR DESCRIPTION
**Pull request by karbates**
Update learning objectives to use Bloom's taxonomy

### CODE REVIEW

File: modules/fhir-community.qmd [hunk 1]
```diff
@@ -13,12 +13,12 @@ roles:
...
-    answer: "You can interact with the FHIR community through [Work Groups](https://confluence.hl7.org/pages/viewpage.action?pageId=4489802), Zulip ([https://chat.fhir.org](https://chat.fhir.org)), and [Connectathons](https://confluence.hl7.org/display/FHIR/Connectathons)."
-  - question: "Where can I ask FHIR related questions?"
-    answer: "Zulip ([chat.fhir.org](https://chat.fhir.org)), the [FHIR Community Forum](http://community.fhir.org), and [StackOverflow](https://stackoverflow.com/questions/tagged/hl7_fhir) are great places to ask FHIR-related questions."
-  - question: "Where can I find more information on HL7?"
-    answer: "HL7 resources are primarily found in [HL7 Confluence](https://confluence.hl7.org) and project status can be seen in [HL7 Jira](https://jira.hl7.org/secure/Dashboard.jspa)."
+  - question: Describe three ways to get involved or seek assistance from the FHIR community.
+    answer: Three ways to engage with the FHIR community are through [Work Groups](https://confluence.hl7.org/pages/viewpage.action?pageId=4489802), Zulip ([https://chat.fhir.org](https://chat.fhir.org)), and [Connectathons](https://confluence.hl7.org/display/FHIR/Connectathons).
```
Comment (masnick) [inline modules/fhir-community.qmd]:
```suggestion
    answer: Options for engaging with the FHIR community include [HL7 Work Groups](https://confluence.hl7.org/pages/viewpage.action?pageId=4489802), Zulip (<https://chat.fhir.org>), and [HL7 Connectathons](https://confluence.hl7.org/display/FHIR/Connectathons).
```

File: modules/fhir-community.qmd [hunk 2]
```diff
@@ -13,12 +13,12 @@ roles:
...
+  - question: Explain the purpose of [chat.fhir.org](https://chat.fhir.org) and give an example of a stream found on the site.
+    answer: Zulip ([chat.fhir.org](https://chat.fhir.org)) is a great places to ask FHIR-related questions. One popular stream for developers is [#implementers](https://chat.fhir.org/#narrow/stream/179166-implementers).
+  - question: Contrast the purpose of HL7’s Confluence site with HL7’s Jira site.
+    answer: HL7 resources and Work Groups are primarily found in [HL7 Confluence](https://confluence.hl7.org), while project statuses and ballot comments can be seen in [HL7 Jira](https://jira.hl7.org/secure/Dashboard.jspa).
```
Comment (masnick) [inline modules/fhir-community.qmd]: I think these are too low-level. I'm happy just deleting these as the general idea is covered by the first learning objective, but feel free to switch to something else if you want.

File: modules/fhir-from-1000-ft.qmd [hunk 3]
```diff
@@ -8,12 +8,12 @@ roles:
...
-  - question: What is the role of resources in FHIR?
+  - question: Summarize the role of resources in FHIR.
     answer: Resources are categories of data in FHIR, and are used to store and retrieve a specific type of data in a pre-defined structure.
-  - question: How can FHIR be customized, and why is this important?
+  - question: Describe how FHIR can be customized and why it is important.
     answer: FHIR can be customized through a process called profiling, where FHIR resources are constrained and extended. This allows FHIR to be used for a wide variety of use cases, including applications that were not imagined by FHIR's creators.
-  - question: What is the FHIR API?
-    answer: It is a standard way for systems to request and send instances of resources to each other.
+  - question: Explain the purpose of the FHIR API.
+    answer: A FHIR API is a standard way for systems to request and send instances of resources to each other.
```
Comment (masnick) [inline modules/fhir-from-1000-ft.qmd]:
```suggestion
    answer: The FHIR API is a standard way for systems to request and send instances of resources to each other.
```

File: modules/key-fhir-resources.qmd [hunk 4]
```diff
@@ -12,11 +12,11 @@ roles:
   - Clinician Scientist/Trainee
 
 learning_objectives:
-  - question: What are three key FHIR resources?
+  - question: Give three examples of key FHIR resources.
     answer: Patient, Observation, and Bundle.
-  - question: How are FHIR resources defined?
+  - question: Explain the significance of a FHIR structure table for a resource and what is included in the table.
```
Comment (masnick) [inline modules/key-fhir-resources.qmd]:
```suggestion
  - question: Explain the significance of a FHIR resource's "structure table" in the FHIR specification and identify what is included in the table.
```

File: modules/key-fhir-resources.qmd [hunk 5]
```diff
@@ -12,11 +12,11 @@ roles:
...
     answer: FHIR resources are defined in a structure table that lists out each attribute's name, flags, cardinality, type, constraints, and descriptions.
```
Comment (masnick) [inline modules/key-fhir-resources.qmd]:
```suggestion
    answer: The data elements of each FHIR resource are listed in a "structure table" on that resource's page in the FHIR specification. The structure table includes each data element's name, flags, cardinality, type, constraints, and descriptions.
```

File: modules/key-fhir-resources.qmd [hunk 6]
```diff
@@ -12,11 +12,11 @@ roles:
...
-  - question: How do FHIR resources support interoperability?
+  - question: Describe how FHIR resources support interoperability.
     answer: A FHIR resource's attributes are rigorously defined, which allows systems to transfer medical information without loss of meaning.
```
Comment (masnick) [inline modules/key-fhir-resources.qmd]:
```suggestion
    answer: The data elements of each FHIR resource are rigorously defined by the FHIR specification. Systems that (correctly) implement the FHIR specification are therefore able have a consistent location and format for storing data, which supports transfer of medical information without loss of meaning.
```

File: modules/smart-on-fhir-intro.qmd [hunk 7]
```diff
@@ -9,9 +9,9 @@ roles:
   - Clinician Scientist/Trainee
 
 learning_objectives:
-  - question: What characteristic of research suggests SMART on FHIR may be helpful?
-    answer: Research that requires integration with an EHR may benefit from SMART on FHIR.
-  - question: What has SMART on FHIR already been used for in published research?
+  - question: Restate how SMART on FHIR supports research with data in EHRs.
```
Comment (masnick) [inline modules/smart-on-fhir-intro.qmd]:
```suggestion
  - question: Explain how SMART on FHIR supports research with data in EHRs.
```

File: modules/smart-on-fhir-intro.qmd [hunk 8]
```diff
@@ -9,9 +9,9 @@ roles:
...
+    answer: SMART on FHIR can help researchers work with multiple institutions' EHRs with a build once and repurpose application.
```
Comment (masnick) [inline modules/smart-on-fhir-intro.qmd]:
```suggestion
    answer: SMART on FHIR allows for a single application to integrate with multiple institutions' EHRs with minimal institution-specific customization. Without SMART on FHIR, substantial institution-specific development work would be needed.
```

File: modules/terminology.qmd [hunk 9]
```diff
@@ -13,11 +13,11 @@ roles:
   - Clinician Scientist/Trainee
 
 learning_objectives:
-  - question: What key role does standard terminology play in enabling health data interoperability?
+  - question: Explain how terminology enables health data interoperability.
     answer: Standard terminology allows data to move across systems while maintaining shared meaning.
-  - question: What are some use cases for terminology in research?
+  - question: Give two examples of use cases for terminology in research.
```
Comment (masnick) [inline modules/terminology.qmd]:
```suggestion
  - question: Identify use cases for terminology in research.
```

File: modules/terminology.qmd [hunk 10]
```diff
@@ -13,11 +13,11 @@ roles:
...
     answer: Use cases include helping researchers automatically build cohorts and aggregating data across different systems or institutions.
-  - question: What are the key FHIR resources related to terminology?
+  - question: Contrast CodeSystems, ValueSets, and ConceptMaps.
```
Comment (masnick) [inline modules/terminology.qmd]:
```suggestion
  - question: Understand how CodeSystems, ValueSets, and ConceptMaps are used in FHIR.
```

File: modules/title-block.html [hunk 11]
```diff
@@ -39,7 +39,7 @@ <h1 class="title">$title$</h1>
 </div>
 </div>
 <div class="callout-body-container callout-body">
-<p class="prompt">After reading this module, you will be able to answer the following questions:</p>
+<p class="prompt">Use the following prompts to gauge your understanding after reading the module:</p>
```
Comment (masnick) [inline modules/title-block.html]:
```suggestion
```

We're going to omit this entirely.

File: modules/fhir-accessing-data.qmd [hunk 12]
```diff
@@ -7,7 +7,7 @@ roles:
 
 learning_objectives2:
   - Understand the basics of how to access data via the FHIR API.
-  - Describe some common types of searches to read data from a FHIR server for reserach.
+  - Describe some common types of searches to read data from a FHIR server for research.
```
Comment (masnick) [inline modules/fhir-accessing-data.qmd]:
```suggestion
  - Describe common types of searches to read data from a FHIR server for research.
```

File: modules/fhir-community.qmd [hunk 13]
```diff
@@ -13,12 +13,12 @@ roles:
   - Clinician Scientist/Trainee
 
 learning_objectives:
-  - question: "How can I get involved with the FHIR community?"
-    answer: "You can interact with the FHIR community through [Work Groups](https://confluence.hl7.org/pages/viewpage.action?pageId=4489802), Zulip ([https://chat.fhir.org](https://chat.fhir.org)), and [Connectathons](https://confluence.hl7.org/display/FHIR/Connectathons)."
-  - question: "Where can I ask FHIR related questions?"
-    answer: "Zulip ([chat.fhir.org](https://chat.fhir.org)), the [FHIR Community Forum](http://community.fhir.org), and [StackOverflow](https://stackoverflow.com/questions/tagged/hl7_fhir) are great places to ask FHIR-related questions."
-  - question: "Where can I find more information on HL7?"
-    answer: "HL7 resources are primarily found in [HL7 Confluence](https://confluence.hl7.org) and project status can be seen in [HL7 Jira](https://jira.hl7.org/secure/Dashboard.jspa)."
+  - question: Describe three ways to get involved or seek assistance from the FHIR community.
```
Comment (masnick) [inline modules/fhir-community.qmd]:
```suggestion
  - question: Describe how to get involved or seek assistance from the FHIR community.
```

File: modules/fhir-from-10000-ft.qmd [hunk 14]
```diff
@@ -17,11 +17,11 @@ roles:
...
 
 learning_objectives:
-  - question: What role does FHIR play in health data interoperability?
+  - question: Summarize the role of FHIR in health data interoperability.
     answer: FHIR is the ubiquitous specification governing interoperability of health systems. This is driven by regulation in the US and FHIR's pragmatic implementation-first approach.
-  - question: Why is FHIR relevant for research?
+  - question: Explain FHIR’s relevance to research.
     answer: At a high level, FHIR is well-adopted throughout the US healthcare system thanks to ONC and CMS rules. FHIR's openness, active development with an implementer-first approach, and extensibility make it a flexible tool for research.
-  - question: What are areas where FHIR can enhance research?
+  - question: Describe ways in which FHIR can enhance research.
```
Comment (masnick) [inline modules/fhir-from-10000-ft.qmd]:
```suggestion
  - question: Describe how FHIR can enhance research.
```

File: modules/fhir-history.qmd [hunk 15]
```diff
@@ -14,11 +14,11 @@ roles:
...
 
 learning_objectives:
-  - question: What are HL7 V2's key limitations?
+  - question: Describe the key limitations of HL7 V2.
     answer: HL7 V2's data format is outdated compared to modern formats like JSON and XML. It also does not support many health-related data types and is not structured enough to support interoperability.
-  - question: What are HL7 V3's key limitations?
+  - question: Describe the key limitations of HL7 V3.
     answer: V3 is structured enough to support interoperability but is difficult to implement and has not been widely adopted.
-  - question: What characteristics of FHIR have led to its success compared to V2 and V3?
+  - question: Contrast the characteristics of FHIR to HL7 V2 and V3. Describe how those characteristics influenced its success over the earlier versions.
```
Comment (masnick) [inline modules/fhir-history.qmd]:
```suggestion
  - question: Understand how FHIR  is related to HL7 V2 and V3, and how the characteristics of these standards influenced their adoption.
```

File: modules/fhir-research.qmd [hunk 16]
```diff
@@ -9,7 +9,7 @@ roles:
   - Clinician Scientist/Trainee
 
 learning_objectives:
-  - question: How have researchers used FHIR in the past?
+  - question: Give an example of how FHIR has been used by researchers.
```
Comment (masnick) [inline modules/fhir-research.qmd]:
```suggestion
  - question: Identify examples of how FHIR has already been used by researchers.
```

File: modules/key-fhir-resources.qmd [hunk 17]
```diff
@@ -12,11 +12,11 @@ roles:
   - Clinician Scientist/Trainee
 
 learning_objectives:
-  - question: What are three key FHIR resources?
+  - question: Give three examples of key FHIR resources.
```
Comment (masnick) [inline modules/key-fhir-resources.qmd]:
```suggestion
  - question: Identify key FHIR resources.
```

**Review by masnick — CHANGES_REQUESTED** [review]
I tried to replace prompts like "List 2 ..." with "Identify ..." or "Understand ..." throughout.

File: modules/fhir-community.qmd (hunk 2 above)
Comment (karbates) [reply modules/fhir-community.qmd]: Good point. I removed the last two questions and submitted new commit with those changes.

**Review by masnick — APPROVED** [review]

**Pull request merged**